webapp/src/shared/components/Snackbar/Snackbar.css

```diff
@@ -0,0 +1,57 @@
+.snackbar {
+  position: fixed;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  border: none;
+  border-radius: 0.75rem;
+  height: 3rem;
+  width: 15rem;
```

Comment:
why forced to 15 wide? We may need to display slightly longer texts, don't you think?

With long texts, it doesn't really work properly.
![Screenshot from 2022-11-10 01-35-02](https://user-images.githubusercontent.com/2774420/200964959-b5ddbb25-bdf9-4159-9a87-c26d09362195.png)

Reply:
Update it to grow with content, thanks :+1: